client-app/shared/catalog/components/category/category-horizontal-filters.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <ProductsFilters
+    class="category-horizontal-filters"
+    orientation="horizontal"
+    :keyword="keywordQueryParam"
+    :filters="filters"
+    :loading="loading"
+    @change="$emit('applyFilters', $event)"
+  >
+    <template #prepend>
+      <VcButton
+        v-if="!hideAllFilters"
+        size="sm"
+        variant="outline"
+        class="category-horizontal-filters__show-all"
+        @click="$emit('showPopupSidebar')"
+      >
+        <template #prepend>
+          <VcIcon name="filter" class="category-horizontal-filters__show-all-icon" />
+        </template>
+        {{ $t("common.buttons.all_filters") }}</VcButton
+      >
+
+      <VcDropdownMenu
+        v-if="!hideSorting"
+        :offset-options="4"
+        class="category-horizontal-filters__sorting"
+        max-height="20rem"
+      >
+        <template #trigger>
+          <VcButton size="sm" variant="outline" class="category-horizontal-filters__sorting-trigger">
+            <template #prepend>
+              <VcIcon name="switch-vertical" class="category-horizontal-filters__sorting-trigger-icon" />
+            </template>
+            {{ $t("common.buttons.sort_by") }}</VcButton
+          >
+        </template>
+        <template #content="{ close }">
+          <div class="py-2">
+            <VcMenuItem
+              v-for="sortingOption in PRODUCT_SORTING_LIST"
+              :key="sortingOption.id"
+              class="category-horizontal-filters__sorting-list"
```

Comment:
Why menu-items class=... __sorting-list?

Reply:
changed name to __sorting-item and the next one to __sorting-input accordingly